transforms/universal/pdf2md/python/.dockerignore

```diff
@@ -0,0 +1,6 @@
+venv/
```

Comment:
Do you need to ignore HF_CACHE folder you reference in the help text for the transform?

Reply:
the huggingface cache is usually in the home folder, not inside the repo, so I don't think it is needed